### ISSUE TITLE
Add a recurring recipe to the tool's creation.

### ISSUE THREAD

**Issue by PinkYuDeer** [GTNewHorizons/GT-New-Horizons-Modpack#17431]
### Your GTNH Discord Username

PinkYuDeer

### Your Pack Version

2.7.0

### Your Proposal

Add a constantly repeating recipe for GT tools at the front of the NEI, 

### Your Goal

similar to "different wood planks make stick". 

### Your Vision

For a long time after I started playing the modpack, I was confused by the toolmaking in NEI. Today, I saw another streamer getting same confused. For new players, "I can't see any of these advanced materials, how do I make a Hammer?" can be improved.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Yamnasm** [comment]
The "scrolling" alternative ingredients like your stick example works only because a stick does not have individual properties from being crafted by different recipies:
![javaw_pIkFWNpKE5](https://github.com/user-attachments/assets/c470522a-5924-4c48-b0a4-aef1b0a71fef)

GT Material Tools have different durabilities and properties depending on the material(s) used to craft them, which makes a "repeating" recipe difficult to handle in a simple NEI tab like the wood stick example.

**Comment by PinkYuDeer** [comment]
Yes, I understand. Different attributes are indeed difficult to display on a single page. So, can we create a "general recipe" that always displays the same type of material for the first or first few recipes, and create a tool with an attribute of "NULL", "undefined", or "material-dependent" to indicate the general?

**Comment by PinkYuDeer** [comment]
Or perhaps arranging the properties of the tools in order could also be a solution. In this way, the confusion caused by lower-level materials appearing first would be reduced.

**Comment by PinkYuDeer** [comment]
Additionally, regarding sorting: I actually can't count the number of times I've searched the inner interface of the fluid extractor for recipes to extract fluids from "metal powder" or "metal ingots." I would be eternally grateful if I could achieve a top-of-the-page display of the powder and pill formula during the fluid extraction interface.

**Comment by chochem** [comment]
maybe you saw those comments before the recipe search was added. This seems resolved.
![image](https://github.com/user-attachments/assets/5d0a94f2-2e52-41d0-bb26-0ae6fe341a9d)

**Comment by PinkYuDeer** [comment]
WOW. I just realised there was a search. That's so cool.

**Comment by Ruling-0** [comment]
I'd definitely argue the search resolves this.
